src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.Linux.cs

```diff
@@ -213,21 +213,8 @@ protected internal override void Write(int pinNumber, PinValue value)
         }
 
         private string ConvertPinValueToSysFs(PinValue value)
-        {
-            string result = string.Empty;
-            switch (value)
-            {
-                case PinValue.High:
-                    result = "1";
-                    break;
-                case PinValue.Low:
-                    result = "0";
-                    break;
-                default:
-                    throw new ArgumentException($"Invalid pin value {value}.");
-            }
-            return result;
-        }
+
+            => value == PinValue.High ? "1" : "0";
```

Comment:
Can't we just cast to int in this case now and return toString?

Reply:
Yes, but that would have to allocate.